evalml/pipelines/time_series_classification_pipelines.py

```diff
@@ -113,14 +132,15 @@ def predict_proba(self, X, y=None):
         y = _convert_woodwork_types_wrapper(y.to_series())
         y_encoded = self._encode_targets(y)
         features = self.compute_estimator_features(X, y_encoded)
-        proba = self.estimator.predict_proba(features.dropna(axis=0, how="any"))
+        features_no_nan, y_encoded = drop_rows_with_nans(features, y_encoded)
+        proba = self._estimator_predict_proba(features_no_nan, y_encoded)
         proba.columns = self._encoder.classes_
         return pad_with_nans(proba, max(0, features.shape[0] - proba.shape[0]))
 
     def _compute_predictions(self, X, y, objectives):
         """Compute predictions/probabilities based on objectives."""
-        y_predicted = None
-        y_predicted_proba = None
+        y_predicted = pd.Series([])
+        y_predicted_proba = pd.DataFrame([])
```

Comment:
I may be confused but why this change? This'll always return a valid prediction value, even if its empty?

Reply:
`drop_rows_with_nans` (which is called in `score`) needs a pandas data type and not `None`. I changed `drop_rows_with_nans` to accept None and so I changed these back to `None`.